src/main/java/org/littil/api/coordinates/service/SearchService.java

```diff
@@ -0,0 +1,17 @@
+package org.littil.api.coordinates.service;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.QueryParam;
+import java.util.Set;
+
+@RegisterRestClient
+public interface SearchService {
+
+    @GET
+    Set<Coordinates> getCoordinatesByAddress(
+            @QueryParam("postalcode") String postalCode,
```

Comment:
Can we use postal_code or postal-code as query param since it's two words?

Reply:
This is indeed possible, it's not postal_code